microsoft-edge/webview2/release-notes.md

```diff
@@ -303,23 +341,7 @@ n/a
 
 * Fixed a bug where the `CoreWebView2EnvironmentOptions.Language` property doesn't change the `accept-language` HTTP header.  (SDK-only)  ([Issue #3635](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3635))
 * Added support for longer runtime installation paths.  (SDK-only)
-* The custom URI scheme registration API now works in WinRT.  (SDK-only):
-
-##### [.NET/C#](#tab/dotnetcsharp)
-
-* [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.customschemeregistrations?view=webview2-dotnet-1.0.2106-prerelease&preserve-view=true)
-
-##### [WinRT/C#](#tab/winrtcsharp)
-
-* [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.2106-prerelease&preserve-view=true#customschemeregistrations)
-
-##### [Win32/C++](#tab/win32cpp)
-
-* [ICoreWebView2EnvironmentOptions4](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.2106-prerelease&preserve-view=true)
-   * [ICoreWebView2EnvironmentOptions4::GetCustomSchemeRegistrations](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.2106-prerelease&preserve-view=true#getcustomschemeregistrations)
-   * [ICoreWebView2EnvironmentOptions4::SetCustomSchemeRegistrations](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.2106-prerelease&preserve-view=true#setcustomschemeregistrations)
-
----
+* The custom URI scheme registration API now works in WinRT.  See the `CustomSchemeRegistrations` property above.  (SDK-only)

```

Comment:
I'm a little confused about the change here

Reply:
The Sep. Prerelease section
https://learn.microsoft.com/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#102106-prerelease
has two spots that cover the same thing -- both sections give dup but incomplete info, in their two tab-groups.  So, in the Oct Release PR 2841, I merged the two tab-groups together and made a single, complete tab-group; then in the Bug Fix entry, instead of an incomplete tab-group, I point to the now-complete tab group in the Promotions section.

1) Promotions > "For WinRT, options have been added to manage custom scheme registration when creating a CoreWebView2Environment:"  Live article's tab-group has 3 tabs, but only the WinRT tab contains content/links, and that's an incomplete subset of the links.  So I made this tab-group complete.
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/549c43c2-a9c4-4421-96d3-ee3035a5ac49)

2) Bug fixes > "The custom URI scheme registration API now works in WinRT. (SDK-only):"  Live article has 3 tabs, containing an incomplete subset of the links.  I got rid of this tab-group and pointed people to the tab-group that's in the Promotions section.
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/bad56b37-17f8-4c42-aa4c-c586458d619b)

Normally the bug fix section doesn't contain a tab-group; tab-groups are in the Promotions section.  This change conforms to that pattern.